src/main.ts

```diff
@@ -12,7 +12,7 @@ export const MustHaveApiVersion = new SpecificationRule({
 });
 
 export default {
-  name: "example-ruleset",
+  name: "name-of-custom-rules-package",
```

Comment:
This is not relating to this PR per-se, but I was thinking if in our default we do like a `const name = 'ruleset-name';` up top we could then do the exported name and this name as that variable and kind of DRY it up and give someone only one spot to update.